sale_fully_invoiced/models/sale.py

```diff
@@ -0,0 +1,35 @@
+from odoo import api, fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    # Same as invoice_status, but excluding invoices in draft
+    invoice_status_validated = fields.Selection(
+        [
+            ("no", "Nothing to Invoice"),
+            ("to invoice", "Can be Invoiced"),
+            ("invoiced", "Fully Invoiced"),
+        ],
+        string="Posted Invoice Status",
+        compute="_compute_invoice_status_validated",
+        store=True,
+        readonly=True,
+        copy=False,
+        default="no",
+    )
+
+    @api.depends("invoice_status", "invoice_ids.state")
+    def _compute_invoice_status_validated(self):
+        # Same as invoice status, execpt it is not invoiced unless there is no invoice
+        # in draft
+        for order in self:
+            if order.invoice_status in ("no", "to invoice"):
+                order.invoice_status_validated = order.invoice_status
+            elif order.invoice_status in ("no", "upselling"):
```

Comment:
I'm quite bugging on this line...

If the order status is "no" on the first `if`, then, the `invoice_status_validated`== `order.invoice_status`== `no`

So, the `elif`should be `elif order.invoice_status == "upselling"` 

No ?